vms/platformvm/txs/executor/standard_tx_executor_test.go

```diff
@@ -1734,7 +1530,7 @@ func TestStandardExecutorRemoveSubnetValidatorTx(t *testing.T) {
 				env := newValidRemoveSubnetValidatorTxVerifyEnv(t, ctrl)
 
 				// Set dependency expectations.
-				env.state.EXPECT().GetTimestamp().Return(env.latestForkTime)
+				env.state.EXPECT().GetTimestamp().Return(env.latestForkTime).AnyTimes().AnyTimes()
```

Comment:
Was this change intended? It seems like we should at least remove one of the `AnyTimes()` calls here.

Reply:
Mixed up manual and "scripted" changes. Dropped